deployment/ccip/view/v1_6/rmnhome.go

```diff
@@ -83,59 +84,19 @@ type RMNHomeSourceChain struct {
 }
 
 type RMNHomeNode struct {
-	PeerId            [32]byte `json:"peerId"`
-	OffchainPublicKey [32]byte `json:"offchainPublicKey"`
-}
-
-func (n RMNHomeNode) MarshalJSON() ([]byte, error) {
```

Comment:
no need for custom marshal unmarshal as we are saving decode peerids and []byte type already marshals correctly to `hex.EncodeToString`